pkgs/development/python-modules/openstep-plist/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+, git
+, pytestCheckHook
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "openstep_plist";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "KO4sGKjod5BwUFQ1MU2S1dG0DyiJ06mdroMbRDsugBk=";
+    extension = "zip";
+  };
+
+  nativeBuildInputs = [ setuptools_scm git cython];
```

```suggestion
  nativeBuildInputs = [ setuptools-scm cython];
```
Whatever git is doing, we want to patch that. I couldn't find a usage in code on the quick.